README.asciidoc

```diff
@@ -201,3 +201,62 @@ Run individual tests by specifying them explicitly:
 ----
 make check TESTS=23-baseclass.t
 ----
+
+How to run test cases
+---------------------
+
+This following instructions shows how to run test cases using the openSUSE test
```

Comment:
I would prefer if you describe the much simpler os-autoinst-distri-example but I don't mind if we mention the openSUSE tests as a real-life example :)

Reply:
I added both examples.

Reply:
Well, thanks for the effort but you misread. I meant in particular https://github.com/os-autoinst/os-autoinst-distri-example , not the openQA-in-openQA tests. The main reason being that https://github.com/os-autoinst/os-autoinst-distri-example has no separate needle git repo which is a test distribution specific setup that is not necessary in general.

Reply:
I've included a link to the minimum example as well.